pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/index/mutable/MutableDictionary.java

```diff
@@ -48,4 +48,11 @@ default int insertionIndexOf(String stringValue) {
     // This method should not be called for unsorted dictionary.
     throw new UnsupportedOperationException();
   }
+
+  /**
+   * This method returns a boolean denoting whether the mutable dictionary can consume any more rows or not.
```

Comment:
Make it clear by saying "Return true if the mutable dictionary can consume an additional row"